testing/protokt-generation/src/test/kotlin/protokt/v1/testing/Edition2023PresenceTest.kt

```diff
@@ -61,44 +61,87 @@ class Edition2023PresenceTest {
 
     @Test
     fun `file with explicit presence and no field features has correct behavior on primitive`() {
-        assertThat(TestFileImplicit::class.propertyIsMarkedNullable("foo")).isFalse()
-        assertThat(TestFileImplicit {}.foo).isEqualTo(0)
+        assertThat(TestFileExplicit::class.propertyIsMarkedNullable("foo")).isTrue()
```

Comment:
This test was broken due to bad copypasta of the message type being asserted on.